.eslintrc.json

```diff
@@ -196,10 +196,6 @@
       "files": ["*.ts", "*.tsx"],
       "extends": ["plugin:@nrwl/nx/typescript"]
     },
-    {
-      "files": ["*.ts"],
-      "extends": ["plugin:ngrx/component-store-strict"]
```

Comment:
suggestion: looks like the `ngrx/component-store-strict` can be replaced with `@ngrx/component-store-strict` and things will just work ™ as before :) Can we try it?

Reply:
It's seems that it was that easy, good spotted :)